cluster/ci/config/prow/labels.yaml

```diff
@@ -402,6 +402,71 @@ repos:
         name: area/webpages
         target: both
         addedBy: label
+      - color: 00D8FF
```

Comment:
Looks like this should be under a stanza just for the console repo?

```suggestion
  openshift/console:
    labels:
      - color: 00D8FF
```

Reply:
whoops. I'll update